third_party/intel/lib/TritonIntelGPUToLLVM/Utility.cpp

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "mlir/Dialect/GPU/IR/GPUDialect.h"
+
 #include "Utility.h"
 #include "intel/include/Dialect/TritonIntelGPU/IR/LinearLayoutConversions.h"
 #include "intel/include/Dialect/TritonIntelGPU/Transforms/Utility.h"
```

Comment:
```suggestion
#include "Utility.h"

#include "mlir/Dialect/GPU/IR/GPUDialect.h"

#include "intel/include/Dialect/TritonIntelGPU/IR/LinearLayoutConversions.h"
#include "intel/include/Dialect/TritonIntelGPU/Transforms/Utility.h"
```

Keep `Utility.h` on top